packages/console/src/components/ItemPreview/index.module.scss

```diff
@@ -0,0 +1,22 @@
+@use '@/scss/underscore' as _;
+
+.item {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+
+  > div:not(:first-child) {
+    margin-left: _.unit(4);
+  }
+
+  .title {
+    font: var(--font-body);
+    color: var(--color-primary);
+  }
+
+  .subtitle {
+    margin-top: _.unit(0.5);
```

Comment:
I think we do need to re-think about this unit thing... I thought 1 unit was the minimum value

Reply:
1 would be the minimum, this is a corner-case. i can change to `2px` if needed

Reply:
will remove it soon